tests/chainer_tests/training_tests/extensions_tests/test_logreport.py

```diff
@@ -0,0 +1,60 @@
+import shutil
+import tempfile
+import unittest
+
+import mock
+
+from chainer import testing
+from chainer import training
+from chainer.training import extensions
+
+
+class TestLogReport(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp_dir)
+
+    def test_trigger(self):
```

Comment:
Sorry, I read my comment again and found that the sentence was inconsistent. My intension was two-fold:

* Write a unit test that directly calls `log_report._write` with expected arguments (without mock object)
* Check appropriate arguments are correctly fed to the the mock object of `_write` method.

You did the second one already. Could you the first one?